objgraph.py

```diff
@@ -303,7 +307,7 @@ def growth(limit=10, peak_stats={}, shortnames=True, filter=None):
     .. versionadded:: 3.3.0
 
```

Comment:
I think this needs a `.. versionchanged:: 3.6.0 added gc_collect_gen parameter` like the other ones.